app/models/expression_file_info.rb

```diff
@@ -11,17 +11,16 @@ class ExpressionFileInfo
 
   # note that species and reference genome/annotation live at the study_file level, not here
 
-  UNITS_VALUES = [nil, 'UMI-corrected raw counts', 'raw counts']
-  validates :units, inclusion: {in: UNITS_VALUES}
+  UNITS_VALUES = ['UMI-corrected raw counts', 'raw counts']
+  validates :units, inclusion: {in: UNITS_VALUES}, allow_blank: true
 
-  BIOSAMPLE_INPUT_TYPE_VALUES = [nil, 'whole cell', 'single nuclei', 'bulk']
-  validates :biosample_input_type, inclusion: {in: BIOSAMPLE_INPUT_TYPE_VALUES}
+  BIOSAMPLE_INPUT_TYPE_VALUES = ['whole cell', 'single nuclei', 'bulk']
```

Comment:
Question about user-facing term consistency:

Does "biosample input type" differ from the newly-optional `biosample_type` field in the metadata convention?  If so, how?  This seems like a potential source of user confusion.

Perhaps we should omit "input" from this field name throughout our code.

Reply:
The "biosample_type" and "biosample_input_type" are distinct. The biosample_type describes the context of the biosample. The biosample can then be bulk processed or processed as whole cell or single nuclei. 

The "biosample_input_type" is determined by "biosample_type" + an experimental processing activity.